mlflow/deployments/cli.py

```diff
@@ -237,6 +236,8 @@ def run_local(flavor, model_uri, target, name, config):
 
 
 def predictions_to_json(raw_predictions, output):
+    from mlflow.pyfunc.scoring_server import _get_jsonable_obj
```

Comment:
This is kind of a bummer, as it seems to imply that our deployments CLI will never support the `predict()` function unless the complete MLflow platform is installed. I think the hope was that users could install deployment plugins alongside the MLflow skinny client and continue to leverage them without also having to fetch the full platform.

Can we factor out `get_jsonable_obj` into a utility that's independent of `mlflow.pyfunc.scoring_server` and introduce some skinny client test coverage for the use of deployment plugins? E.g. I think we can run this: https://github.com/mlflow/mlflow/blob/master/tests/test_cli.py

Reply:
Refactoring it is possible but it won't allow us to run the test_cli.py given it's dependence on flask.

__init__.py also depends on flask and would cause an error.

This would require more changes than move get_jsonable_obj to have a validated test.

Where would we want to move get_jsonable_obj?

Reply:
Sounds like we can test everything in `test_cli` except for `test_run_local`, right? In that case, perhaps we can move `test_run_local` into its own test suite?

I think `proto_json_utils` seems like a pretty reasonable place to define the helper function.